style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
+* Prefer `===` and `!===` to `==` and `!=`.
+* Avoid using arrow functions when not necessary.
```

Comment:
Is there any more guidance you can can provide to clarify "when necessary" or why to avoid? Is it just for binding `this` with fat arrow?